contracts/RouterV2.sol

```diff
@@ -242,7 +244,6 @@ contract Router {
         return swapBaseToSynthFM(inputAmount,  synthIN,  msg.sender);
     }
     function swapBaseToSynthFM(uint inputAmount, address synthOUT, address member) public returns (uint output){
-         require(iSYNTHFACTORY(_DAO().SYNTHFACTORY()).isSynth(synthOUT) == true, "!synth");
```

Comment:
This check is inside the Pool's mintSynths() function; no need to double-check the same thing (save gas and refactor)